main.go

```diff
@@ -67,7 +67,7 @@ var (
 
 	// Runtime behaviour flags
 	syslogUseCurrentYear = flag.Bool("syslog_use_current_year", true, "Patch yearless timestamps with the present year.")
-	overrideTimezone     = flag.String("override_timezone", "", "If set, use the provided timezone in timestamp conversion, instead of the local zone.")
+	overrideTimezone     = flag.String("override_timezone", "Local", "If set, use the provided timezone in timestamp conversion, instead of the local zone.")
```

Comment:
As noted, please leave the value but update the description, and optionally add a comment to the docs.  (I can do this if you aren't sure where to.)